### DISCUSSION TITLE
Importing .reg file the right way

### DISCUSSION THREAD

**Discussion by crashtestman** [WixEdit/WixEdit#30] [Q&A] [answered]
Hello,

Correct me if I'm wrong but I'm aware that it is possible to import ".reg" file so that the resulting msi apply all of its content.
Though I'm not sure how to proceed. I tried to Import it a as file, child of SourceDir but that does not seems to be the way to do it.

What would be the procedure to import a .reg file so it can be applied by the resulting msi file ?

Thank you in advance.

**Comment by icnocop** [comment, marked as answer]
This question is not related to WixEdit.

See https://stackoverflow.com/questions/11988805/adding-a-reg-file-to-registry-wix
